python/python/Ice/Slice.py

```diff
@@ -0,0 +1,61 @@
+# Copyright (c) ZeroC, Inc.
+
+import IcePy
+
+
+def compileSlice(args: list[str]) -> int:
+    """
+    Compile Slice definitions into Python code using the Slice for Python compiler (`slice2py`).
+
+    This function behaves exactly like the `slice2py` command-line tool: it processes the provided
+    Slice definitions and generates Python files in the directory specified by the `--output-dir`
+    argument.
+
+    If you want to load Slice definitions directly into the current Python interpreter *without*
+    generating files, use :func:`loadSlice` instead.
+
+    Parameters
+    ----------
+    args : list[str]
+        The list of arguments to pass to the Slice for Python compiler.
+        The accepted arguments are identical to those supported by the `slice2py` command-line tool.
+
+    Returns
+    -------
+    int
+        The exit code returned by the Slice for Python compiler.
+    """
+    return IcePy.compileSlice(["Ice.compileSlice", *args])
```

Comment:
What is this first entry in the list?

Reply:
Currently this is called by both slice2py native compiler, the slice2py Python script, and IcePy.compileSlice.

slice2py pip script just call this and so stderr output and help makes sense there.
slice2py is impleted by calling IcePy.compileSlice

We can add a separate version for use directly in Python script.